packages/libs/ui/nft-react-hooks/src/lib/provider.tsx

```diff
@@ -64,19 +63,21 @@ const client = new ApolloClient({
 });
 
 interface IcyProviderProps {
-  apiKey: string;
+  apiKey?: string;
   children: React.ReactNode;
 }
 
 function IcyProvider(props: IcyProviderProps) {
   const tokenStore = useTokenStore();
 
   useEffect(() => {
-    const apiKey = props.apiKey
+    const { apiKey } = props;
     if (typeof apiKey !== 'string' || apiKey.length === 0) {
-      throw new Error('You must pass a valid API key into <IcyProvider>');
+      console.warn(
+        'nft-react-hooks warning: no apiKey provided, significant rate limits will be applied'
```

Comment:
hmm "significant" sounds very vague. I'd prefer if we could provide an actual limit number if possible

Reply:
cool, this a bit of a premature comment since this is still a draft :) i knew you were on it

Reply:
actually, after some digging on the code, accessing whether the user has passed through an api token or not would require some slightly heavier refactoring adding a token/auth context provider.
Instead I think it'd be better to make the nudge that this is only intended for dev usage on initial startup (0e4d55f406c943ecc8a615b2708901bdb7ada1d2). 
<img width="703" alt="image" src="https://user-images.githubusercontent.com/40798232/182880399-26592417-173b-480f-be04-35bec30c0bec.png">
WDYT @felipecsl @gayleQN @danieljvdm @jkolyer?

Reply:
sounds good to me!

Reply:
```no apiKey provided. Access with no apiKey is heavily rate limited and intended for development use only. For higher rate limits or production usage, create an account...```

Reply:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/40798232/182950037-f11e9b00-f17c-498f-aa2c-3d2d74ce3d91.png">
Updated on a8c83b9!